core/src/main/kotlin/knote/script/PageConfiguration.kt

```diff
@@ -51,13 +51,32 @@ class PageConfiguration : ScriptCompilationConfiguration({
 
         beforeCompiling { context ->
             val reports = mutableListOf<ScriptDiagnostic>()
+            val scriptFile = (context.script as FileScriptSource).file
+            val notebookDir = scriptFile.parentFile.parentFile
+            val notebookId = scriptFile.parentFile.parentFile.name
```

Comment:
Can we do 
```
val notebookId = notebookDir.name
```
here?

Reply:
yes, folders were moved around and some code ended up being duplicated